src/app/screens/Settings.tsx

```diff
@@ -42,8 +42,18 @@ function Settings() {
     setNostrPrivateKey(priv ?? "");
   };
 
+  const checkCameraPermissions = async () => {
+    const devices = await navigator.mediaDevices.enumerateDevices();
+    const cameraDevices = devices
+      .filter((device) => device.kind === "videoinput")
+      .filter((videoDevice) => videoDevice.label !== "");
+    cameraDevices.filter((device) => device.kind === "videoinput");
```

Comment:
Why do you filter for `device.kind === "videoinput"` twice?

And can't you simply do:

```js
devices.filter(device => device.kind === "videoinput" && device.label !== "")
```
!?

Reply:
ah, another problem: this line actually doesn't change `cameraDevices` and you can just delete it. Leftover from debugging!?

Reply:
on my machine I have no device where `device.kind === "videoinput" && device.label !== ""`!?
Checked on the Codepen demo I wrote and where I've given camera access permissions I get:
```js
kind: "videoinput"
label: ""
```
from `navigator.mediaDevices.enumerateDevices()`

Reply:
Looks like firefox has an issue with iframes asking for permissions, I copied the code from your pen and ran it locally (index.html and index.js), and it worked fine:

(before allowing and after allowing:)

<img width="460" alt="Screenshot 2022-12-21 at 12 17 49 PM" src="https://user-images.githubusercontent.com/64399555/208839788-ae3037b0-e21c-458e-9c96-cf691caaaccd.png">

Reply:
> devices.filter(device => device.kind === "videoinput" && device.label !== "")

makes sense!

> Leftover from debugging!?

Oops yes! removing it

Reply:
ah, thanks for rebuilding the codepen locally to check that the camera is there 👍